google/google.go

```diff
@@ -44,15 +44,18 @@ var conf *oauth2.Config
 var state string
 var store sessions.CookieStore
 
+var loginURL string
+
 func randToken() string {
 	b := make([]byte, 32)
 	rand.Read(b)
 	return base64.StdEncoding.EncodeToString(b)
 }
 
 // Setup the authorization path
-func Setup(redirectURL, credFile string, scopes []string, secret []byte) {
+func Setup(redirectURL, cLoginURL string, credFile string, scopes []string, secret []byte) {
```

Comment:
I wouldn't modify this function signature. Doing so will break existing functionality and require a major version increment. Could we maybe use a mutator/setter to set the loginURL, something like `SetLoginURL(loginURL string)`?

Reply:
Or something like `WithLoginURL(s string)`